src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const object = sourceString
+    .split(';')
+    .map((element) => {
+      return element.split(':').map((part) => part.trim());
+    })
+    .filter((array) => array.length !== 1)
```

Comment:
The variable name 'array' in the filter callback is not descriptive. It represents a single style declaration split into property and value, so a more descriptive name like 'declaration' would be clearer.